src/views/authenticated/components/UpcommingCharge.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
```

Comment:
Component looks very similar to PreviousCharge.tsx make one more generic component which we can use in both cases